distributed/diagnostics/eventstream.py

```diff
@@ -11,6 +11,7 @@ class EventStream(SchedulerPlugin):
     """Maintain a copy of worker events"""
 
     def __init__(self, scheduler=None):
+        self.name = "EventStream"
```

Comment:
w.r.t. https://github.com/dask/distributed/pull/5120#issuecomment-887656555: Here's an example of giving a name to a builtin plugin